README.md

```diff
@@ -57,6 +57,18 @@ Allowed URL Redirects: 'https://app.lara.docker/users/auth/cc_portal_localhost/c
 2. In the Portal, edit `.env` set SITE_URL=https://app.portal.docker (Ensure that its using https. Othewise there will be communication issues between Portal and lara)
 3. You may need to use the rails console in LARA to set the `is_admin` flag to the portal admin user.
 
+### Communication with reporting service
+To communicate with reporting service you need to make sure you update '.env' with below variables
+1. REPORT_SERVICE_URL=https://us-central1-report-service-dev.cloudfunctions.net/api
+2. REPORT_SERVICE_TOKEN as 'devtoken' for development
+3. REPORT_SERVICE_SELF_URL=https://app.lara.docker (for automation https://authoring.dev.docker) 
+4. REPORT_SERVICE_TOOL_ID with lara host name. Example: app.lara.docker  and for automation authoring.dev.docker
```

Comment:
This section should be updated now that the variables the developer needs to set have changed.

Reply:
Sorry, my bad. Fixed now. I also removed the token. I hope it's fine.